frontend/src/components/Cell.tsx

```diff
@@ -28,18 +28,23 @@ export const Cell = ({
         on {`${date}`}
       </label>
       <input
-        type="number"
+        type="text"
         id={`${topic.issue.id}${topic.activity.id}${formatDate(
           date,
           "yyyy-MM-dd"
         )}`}
-        min={0}
         onChange={(event: any) => {
+          //makes sure that users can only input positive numbers up to 999.99999999...
+          //with an unlimited number of decimals behind the delimiter
+          event.target.value = event.target.value
+            .replace(/[^0-9.]/g, "")
+            .replace(/(^\d{3})\d/g, "$1")
```

Comment:
i think we could take a different approach when validating the expression. Instead of replacing the values, we should make sure our regex actually compiles, and base on that give feedback about whether the input is valid or not.

Reply:
> i think we could take a different approach when validating the expression. Instead of replacing the values, we should make sure our regex actually compiles, and then give vidual feedback to the user.

Do you mean to come up with a regular expression that matches exactly the type of number that we expect, and then complain to the user if that expression does not match?

```
^[1-9]\d{0,2}(.\d{1,2})?$
```